docker/z-quantum-default/Dockerfile

```diff
@@ -63,7 +61,7 @@ RUN python3 -m pip install --no-cache git+git://github.com/zapatacomputing/z-qua
 
 # Misc libraries that we'd like to have already preinstalled.
 RUN python3 -m pip install \
-        codecarbon \
+        codecarbon==1.2.0 \
```

Comment:
Why is this exact pin required?

Reply:
Cause the code that's using it is quite fragile and I'm afraid that changing a different version might accidentally break things.

Reply:
Can you please add a comment to the dockerfile that describes what code exactly depends on this pin?

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-core/pull/432/commits/aa545118c3f1deb3861d9f94d6d122f2726d291a